cloud-client/src/main/java/ClientController.java

```diff
@@ -72,10 +78,13 @@ public void initialize(URL location, ResourceBundle resources) {
                     try {
                         Object obj = is.readObject();
                         LOG.info("Received " + obj.getClass().getName());
-                        switch (obj.getClass().getName()) {
-                            case "TextMessage": LOG.info(obj.toString());
-                                                break;
-                            case "CommandMessage": ProcessCommandResult(obj);
+                        if (obj instanceof TextMessage) {
+                            LOG.info(obj.toString());
+                        } else if (obj instanceof CommandMessage) {
+                            ProcessCommandResult((CommandMessage)obj);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.